### DISCUSSION TITLE
workflow configuration missing

### DISCUSSION THREAD

**Discussion by watsonsimpore** [ionos-deploy-now/ionos-deploy-now#92] [Q&A]
I have a problem with my build workflows configuration.
Indeed on my hosting panel it says workflow configuration missing
someone can help me?

**Comment by ionos-deploy-now-support** [comment]
> I have a problem with my build workflows configuration. Indeed on my hosting panel it says workflow configuration missing someone can help me?

thank you for your message, could you please provide your customer id? im unable to find you with your github user.

**Reply by watsonsimpore** [reply]
This my custumer id:864113448

**Reply by watsonsimpore** [reply]
There is someone here?...

**Reply by Robert95Sch** [reply]
Hi @watsonsimpore , after connecting a repository to Deploy Now, a .github/workflows folder is usually created within your repository. This folder contains yaml files that describe [GitHub actions workflows](https://docs.ionos.space/docs/github-actions-customization/) including the build configuration you have provided in the project setup. It looks like these files have not been created or were overwritten by a later commit. You can deploy or [Laravel Starter](https://github.com/ionos-deploy-now/laravel-starter) as a reference to compare. Could you maybe connect your repository once again and check whether Deploy Now is injecting the workflow files correctly?

**Reply by watsonsimpore** [reply]
I cant deploy again because the deployment is already finished. I contact ionos support they tell me that the problem occurs with the workflow file in github. So i'm dont understand github workflow configuration well! If there is a default workflow file for IONOS deplow-now you can give me that I will put it in my project and re-run!

**Reply by Robert95Sch** [reply]
The problem is indeed the missing workflow file in GitHub that describes how your project should be built by GitHub Actions and deployed to IONOS. All the workflow files should be injected right after the project creation. 

If you delete the project in Deploy Now and set up the project in Deploy Now once again, it should work as expected. If you don't which to delete the projects, you can insert the workflow files similar to these ones: https://github.com/Robert95Sch/laravel-starter-with-workflow/tree/main/.github/workflows

**Reply by watsonsimpore** [reply]
Hello

I followed your advice and tried to create my own workflow file. The build is done very well arrived at the deployment that an error that appears that I can not understand I'll put you the screenshot in file

Thanks to you!

![Capture d'écran 2023-05-13 131831.png](https://github.com/ionos-deploy-now/ionos-deploy-now/assets/113101909/e1296f36-ae7a-44be-8667-991aae98e686)

![Capture d'écran 2023-05-13 133236.png](https://github.com/ionos-deploy-now/ionos-deploy-now/assets/113101909/0230453e-1700-442e-bde6-8eecda6654ee)

**Reply by Robert95Sch** [reply]
Hi @watsonsimpore ,
I saw in your repo that you've figured out that a config.yaml similar to [this one](https://github.com/Robert95Sch/laravel-starter-with-workflow/blob/main/.deploy-now/laravel-starter-with-workflow/config.yaml) is required. However, it looks like there are still problems with the remote commands. Let me know if there's anything I can help you with

**Reply by watsonsimpore** [reply]
How can I resolve the remote commands problems?

**Reply by Ap3rtur3** [reply]
Hey @watsonsimpore, I checked the failed deployment and it seems like the database config contains wrong values. To explain, you cannot create those env variables yourself, they have to be provided by Deploy Now.

Can you try to delete the project in Deploy Now (only in Deploy Now, but not in Github) and recreate it with the Deploy Now UI?
Then your variables in the .env file should be right.

**Comment by watsonsimpore** [comment]
This my custumer id:864113448
